### ISSUE TITLE
Invalid HTTP upgrade" with Swift Client

### ISSUE THREAD

**Issue by RaymondRigil** [miguelgrinberg/Flask-SocketIO#687]
Swift client seems to connect but is unable to pass or view events.  Javascript SocketIO client has no issues seeing/passing events.

The Swift socketio client log showed no other errors.

To investigate further, we then tried using the Starscream library our Swift SocketIO client library is built on, and saw that this was happening:

"Invalid HTTP Upgrade"

The transport protocol used by the javascript library appears to be "polling"

Any ideas as to what may be happening, and how we can get around this?

**Comment by miguelgrinberg** [comment]
Please enable logging on the server to see if that gives any useful information. Add `engineio_logger=True` to your `SocketIO` class for that.

**Comment by RaymondRigil** [comment]
Hi Miguel, thanks for your response.  We enabled logging, removed any forced transport flags in Swift client, and could definitely see events emitted by the Swift client...the only difference was that it was going to the wrong namespace.

So our server url was set up as  'http://x.x.x.x:5080/test' and server was looking for requests on '/test' namespace, and even though this same URL was used in Swift client, the Swift framework was sending it to the '/' namespace instead.

So we solved this by changing the server to use '/' namespace instead of /test and this solved the problem.  The URL used by both Swift and js clients is now 'http://x.x.x.x:5080' instead of 'http://x.x.x.x:5080/test'.

I'm guessing that somehow the Swift framework has a bug that limits us to '/' namespace, or possibly we're not configuring it correctly, although I doubt the latter, as we were passing the right URL (just as in JS client) and I haven't see any other configuration options in the Swift client documentation for namespaces.
